LEGO1/lego/sources/geom/legoedge.cpp

```diff
@@ -4,6 +4,14 @@
 
 DECOMP_SIZE_ASSERT(LegoEdge, 0x24)
 
+// TODO Based on the offset, this should be in the header, but as a stub it's getting inlined when there...
+// STUB: LEGO1 0x1002ddc0
+LegoResult LegoEdge::FUN_1002ddc0(LegoWEEdge* p_face, Vector3& p_point)
```

Comment:
In cases like this it's important to implement the inline function since it very often has a major effect on the code generation of the caller, to the point it's impossible to match the caller (`VTable0x88` in this case) without the right implementation. Creating a non-inline stub has a completely different impact on the code generation than a compiler generated materialization of an inline function

Reply:
Implemented now, in the class header. This has not unexpectedly made the match on 0x88 worse.

Reply:
That's fine - we do know for certain it is an inline function, the match will be figured out eventually. We should not implement things incorrectly just for the sake of the match.

Reply:
Yeah, that is fine - since we know for certain it is an inline function it's better to implement it correctly now and figure out the match later if necessary. I'll take a look at it tomorrow and see if I can make it line up